synapse/storage/databases/state/store.py

```diff
@@ -72,6 +73,26 @@ def __len__(self) -> int:
         return len(self.delta_ids) if self.delta_ids else 0
 
 
+# Return type of state_filter_rough_size_comparator. Must be comparable.
+StateFilterRoughSizeComparator = Tuple[int, int]
```

Comment:
I think this is used only once, so doesn't seem very helpful to have it as a separate type?